doc/source/getting_started/quickstart.rst

```diff
@@ -0,0 +1,98 @@
+.. _quickstart:
+
+==========
+Quickstart
+==========
+
+This concise introduction demonstrates how to estimate Pi in a parallel manner
+using the Monte Carlo method with Xoscar.
+
+We import :code:`xoscar` at the very beginning:
+
+.. code-block:: python
+
+   import asyncio
+   import xoscar as xo
+
+
+Create actor pools
+------------------
+
+To begin, we need to create actor pools, each of which will run within its own individual process.
+
+.. seealso::
+   :ref:`actor-pool`
+
+
+.. code-block:: python
+
+   dop = 4 # degree of parallelism
+   loop = asyncio.get_event_loop()
+   loop.run_until_complete(xo.create_actor_pool(address="localhost:9999", n_process=dop))
```

Comment:
Create a main.py and use "python main.py" to create a pool, how do you think?

Reply:
Sure, but what's the difference between the two approches?

Reply:
There'll be another PR for a simple commandline tool for Xoscar.